src/test/resources/pages/AllJsUnitTests.html

```diff
@@ -3,35 +3,35 @@
+  <script type="text/javascript" src="../../../main/webapp/js/instructorCourseEnrollPage.js"></script>
+  <script type="text/javascript" src="../../../main/webapp/js/instructorCourseDetails.js"></script>
+  <script type="text/javascript" src="../../../main/webapp/js/instructorFeedbacks.js"></script>
+  <script type="text/javascript" src="../../../main/webapp/js/instructorFeedbackEdit.js"></script>
+  <script type="text/javascript" src="../../../main/webapp/js/instructorStudentList.js"></script>
+  <script type="text/javascript" src="../../../main/webapp/js/instructor.js"></script>
+  <script type="text/javascript" src="../../../main/webapp/js/student.js"></script>
+  <script type="text/javascript" src="../../../main/webapp/js/index.js"></script>
 
-  <script type="text/javascript" src="CommonTestFunctions.js"></script>
+  <script type="text/javascript" src="../../javascript/CommonTestFunctions.js"></script>
   <!-- Test scripts -->
-  <script type="text/javascript" src="CommonJsTest.js"></script>
-  <script type="text/javascript" src="InstructorCourseDetailsJsTest.js"></script>
-  <script type="text/javascript" src="InstructorCoursesJsTest.js"></script>
-  <script type="text/javascript" src="InstructorEnrolmentJsTest.js"></script>
-  <script type="text/javascript" src="InstructorEvalJsTest.js"></script>
-  <script type="text/javascript" src="InstructorEvalResultsJsTest.js"></script>
-  <script type="text/javascript" src="InstructorFeedbackEditJsTest.js"></script>
-  <script type="text/javascript" src="InstructorFeedbacksJsTest.js"></script>
-  <script type="text/javascript" src="InstructorStudentListJsTest.js"></script>
-  <script type="text/javascript" src="InstructorJsTest.js"></script>
-  <script type="text/javascript" src="StudentJsTest.js"></script>
-  <script type="text/javascript" src="SubmissionCountJsTest.js"></script>
+  <script type="text/javascript" src="../../javascript/CommonJsTest.js"></script>
+  <script type="text/javascript" src="../../javascript/InstructorCourseDetailsJsTest.js"></script>
+  <script type="text/javascript" src="../../javascript/InstructorCoursesJsTest.js"></script>
+  <script type="text/javascript" src="../../javascript/InstructorEnrolmentJsTest.js"></script>
+  <script type="text/javascript" src="../../javascript/InstructorEvalJsTest.js"></script>
+  <script type="text/javascript" src="../../javascript/InstructorEvalResultsJsTest.js"></script>
+  <script type="text/javascript" src="../../javascript/InstructorFeedbackEditJsTest.js"></script>
+  <script type="text/javascript" src="../../javascript/InstructorFeedbacksJsTest.js"></script>
+  <script type="text/javascript" src="../../javascript/InstructorStudentListJsTest.js"></script>
+  <script type="text/javascript" src="../../javascript/InstructorJsTest.js"></script>
+  <script type="text/javascript" src="../../javascript/StudentJsTest.js"></script>
+  <script type="text/javascript" src="../../javascript/SubmissionCountJsTest.js"></script>
```

Comment:
Related to the newest implementation, `LocalTestFileUrl` (this is the use case of `file:///` URL I was talking about).
With that I'm able to navigate to files inside the test files folder with a method call instead of repeatedly using `... new File(".").getCanonicalPath() ...` (this was repeated 4 times before), but this file `AllJsUnitTests.html` is not inside the test folder so it had to be moved. These are the changes needed for the external files to be referenced correctly.